spec/features/user_management_spec.rb

```diff
@@ -0,0 +1,62 @@
+require './spec/factories/user'
+
+feature 'User sign up' do
+
+
+  scenario 'I can sign up as a new user' do
+    expect { sign_up }.to change(User, :count).by(1)
+    expect(page).to have_content('Welcome, pip')
+    expect(User.first.email).to eq('pip@example.com')
+  end
+
+  scenario ' requires a matching comfirmation password' do
+    expect { sign_up(password_confirmation: 'wrong_password')}.not_to change(User, :count)
+  end
+
+  scenario 'with a password that does not match' do
+  expect { sign_up(password_confirmation: 'wrong') }.not_to change(User, :count)
+  expect(current_path).to eq('/users/new')
+  expect(page).to have_content 'Password does not match the confirmation'
+end
+
+  scenario 'I cannot sign up with an existing email' do
+  user = build :user
+  sign_up_as(user)
+  click_button 'Sign up'
+  sign_up_as(user)
+  expect { click_button('Sign up') }.to change(User, :count).by(0)
+  expect(page).to have_content('Email is already taken')
+end
```

Comment:
Lint/BlockAlignment: `end` at 29, 0 is not aligned with `scenario 'I cannot sign up with an existing email' do` at 22, 2